pkg/build/rpm.go

```diff
@@ -0,0 +1,52 @@
+package build
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+func (b *Builder) getRPMFileNames() error {
+	b.rpmSourceDir = filepath.Join(b.buildConfig.ImageConfigDir, "rpms")
+
+	rpms, err := os.ReadDir(b.rpmSourceDir)
```

Comment:
same

Reply:
For this one, "rpmSourceDir" refers to the user-created configuration directory. In here, I think it makes it significantly easier for us when the user knows to make an RPM dir and put all of their RPMs in there to be copied over to the image, though I'm interested to see other views